cypress/cypress/integration/homepage.spec.js

```diff
@@ -0,0 +1,100 @@
+// homepage.spec.js created with Cypress
```

Comment:
this nested folder of `cypress/cypress/` should not exist. Delete please

Reply:
I had meant to do this - I think it was from choosing the wrong side of a merge.